WcaOnRails/spec/support/database_cleaner.rb

```diff
@@ -1,6 +1,8 @@
 RSpec.configure do |config|
   config.before(:suite) do
+    ActiveRecord::Base.connection.execute("SET foreign_key_checks = 0;")
     DatabaseCleaner.clean_with :truncation
+    ActiveRecord::Base.connection.execute("SET foreign_key_checks = 1;")
```

Comment:
Hmm this wasn't a problem before and I've never seen something like this in database cleaner file. Do we do something special now?